vectordatasource/transform.py

```diff
@@ -4597,6 +4631,24 @@ def _normalize_de_netref(network, ref):
     return network, ref
 
 
+def _normalize_gr_netref(network, ref):
+    ref = _make_unicode_or_none(ref)
+
+    prefix, ref = _splitref(ref)
+    if prefix in (u'ΕΟ', 'EO'):
```

Comment:
If we know that ref is unicode, then doesn't `prefix` also have to be unicode here?

Reply:
Good point! Although it seems to work OK without it, I've fixed them both to be unicode in df0b524, so hopefully it's clearer that `unicode` is being compared with `unicode`.

When are we moving to Python3 again? :wink:

Reply:
I'm glad you put that comment in about the characters looking the same, because it initially looked like a redundant comparison to me. 👍 

> When are we moving to Python3 again? 😉

When python4 comes out :trollface: